src/script/components/ServiceList.tsx

```diff
@@ -25,7 +25,7 @@ import {ParticipantItem} from 'Components/list/ParticipantItem';
 import {KEY} from 'Util/KeyboardUtil';
 
 import type {ServiceEntity} from '../integration/ServiceEntity';
-import {t} from '../util/LocalizerUtil';
+import {t} from '../util/LocalizerUtil/LocalizerUtil';
```

Comment:
```suggestion
import {t} from '../util/LocalizerUtil';
```

There is an `index.ts` that should be sufficient